result.html

```diff
@@ -0,0 +1,171 @@
+                            <div class="quiz-body">
+                                <p>Which of the following is NOT a benefit of using mulch?</p>
+
+                                <div class="quiz-options">
+                                    <input type="radio" name="options" id="first" disabled />
+                                    <label class="options" for="first"> To keep the soil cool </label>
+                                    <input type="radio" name="options" id="second" disabled />
+                                    <label class="options" for="second"> To prevent weeds </label>
+                                    <input type="radio" name="options" id="third" disabled />
+                                    <label class="options" for="third"> To dry out wet soil </label>
+                                    <input type="radio" name="options" id="fourth" disabled />
+                                    <label class="options right" for="fourth"> To prevents bugs </label>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="results">
+                    <span class="number-badge">5</span>
+                    <div class="quiz-container">
+                        <div class="quiz-wrapper">
+                            <div class="quiz-body">
+                                <p>Which of the following is NOT a benefit of using mulch?</p>
+
+                                <div class="quiz-options">
+                                    <input type="radio" name="options" id="first" disabled />
+                                    <label class="options" for="first"> To keep the soil cool </label>
+                                    <input type="radio" name="options" id="second" disabled />
+                                    <label class="options" for="second"> To prevent weeds </label>
+                                    <input type="radio" name="options" id="third" disabled />
+                                    <label class="options" for="third"> To dry out wet soil </label>
+                                    <input type="radio" name="options" id="fourth" disabled />
+                                    <label class="options right" for="fourth"> To prevents bugs </label>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </main>
+        <footer class="footer">
+            <div class="footer_wrapper">
+                <span>Made with <i class="fas fa-heart"></i> by <a href="https://portfolio-alpha-liart.vercel.app/" class="text-gray"> Animesh Garai</a></span>
+                <div class="footer__links">
+                    <a href="https://github.com/animeshgarai09" target="_blank"><i class="fab fa-github"></i></a>
```

Comment:
If you are using `target blank` you should also use `rel` attribute with `noopener noreferrer` value. It helps in securing your site from malicious attacks.

Reply:
Thank you for the insight. will change 👍🏻